source/install/docker-local-machine.rst

```diff
@@ -9,6 +9,8 @@ Note: This configuration should not be used in production, as it's using a known
 
 If you're looking for a production installation with Docker, please see the `Mattermost Production Docker Deployment Guide <http://docs.mattermost.com/install/prod-docker.html>`_.
 
+If you have any problems installing, see the `troubleshooting guide <https://www.mattermost.org/troubleshoot/>`_. To submit an improvement or correction, click Edit at the top of this page.
```

Comment:
See https://docs.mattermost.com/install/install-ubuntu-1604.html for an example of how this note is implemented.